assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
Updated here so it's easier to use: https://github.com/18F/govt-wide-patternlibrary/commit/e05f7d179e5caa657775c09e5782249cdf8eec07